freqtrade/strategy/informative_decorator.py

```diff
@@ -80,12 +80,11 @@ def _create_and_merge_informative_pair(strategy, dataframe: DataFrame, metadata:
         # Not specifying an asset will define informative dataframe for current pair.
         asset = metadata['pair']
 
-    if '/' in asset:
-        base, quote = asset.split('/')
-    else:
-        # When futures are supported this may need reevaluation.
-        # base, quote = asset, ''
-        raise OperationalException('Not implemented.')
+    market = strategy.dp.market(asset)
+    if market is None:
+        raise OperationalException(f'Market {asset} is not available.')
+    base = market['baseId']
+    quote = market['quoteId']
```

Comment:
any particular reason to go with "baseId" instead of "base" (Which we are using already in the exchange class ...) ?

Reply:
I asked about these in the chat. I got a vague answer from you and/or @samgermain that some exchanges may set 'base' as 'Etherium' while having 'baseId' as 'ETH'. Though in that case it would make other freqtrade code using 'base'/'quote' invalid. So i went ahead and dug in ccxt source code.
```js
            const baseId = this.safeString (market, 'baseAssetName');
            const quoteId = this.safeString (market, 'quoteAssetName');
            const id = this.safeString (market, 'marketId');
            const base = this.safeCurrencyCode (baseId);
            const quote = this.safeCurrencyCode (quoteId);
```
Seems like `baseId` is often source of `base` string. I think i can change it to using `base`/`quote` then.

Reply:
You should use `base` and `quote`